sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1505,9 +1529,19 @@ private CaseDataDto caseSave(CaseDataDto dto, boolean handleChanges, Case caze,
 
 		doSave(caze, handleChanges, existingCaseDto);
 
+		if (syncShares && sormasToSormasFacade.isFeatureConfigured()) {
```

Comment:
`isSharingCasesContactsAndSamplesEnabledForUser` is too restricted.
In production someone shares the case and others without share rights can edit it, so it doesn't matter if the  that makes changes to the case has share rights, it must be synced with the other instances

Reply:
My idea was that maybe this feature toggle can be used to stop the sharing or case data immediately for one instance.
I get your point and I think that also makes sense because otherwise you can end up with 2 versions of a case with conflicting information.
I will bring this point up to discussion and for now accept this as it is.